dtests/src/resources/regression_docs/HowToRunRowStoreTests.md

```diff
@@ -0,0 +1,105 @@
+# How To Run RowStore Hydra Tests
+
+## Running the battery tests
+
+#### Pre-requisites:
+
+1. Build the product using ./gradlew clean buildAll
+
+2. Ensure that no java processes are running on the machines
+
+3. Result directory path needs to be already created for storing the logs of test run
+
+**Note:** SnappyData is build using 64-bit JVM and tests are also run using 64-bit JVM.
+
+#### To run the battery test, execute below commands:
+
+```
+export SNAPPYDATA_SOURCE_DIR=<product checkout dir>
+export JTESTS=$SNAPPY_HOME/store/tests/sql/build-artifacts/linux/classes/main
+$SNAPPYDATA_SOURCE_DIR/sample-runbt.sh <result-directory-path> <snappydata-base-directory-path> [-l <local-conf-file-path> -r <num-to-run-test> -m <mailAddresses>] <list-of-bts>
```

Comment:
-r <num-to-run-test> needs to be corrected. Its numTimes to run batterytest. Also <snappydata-base-directory-path> should be replaced with $SNAPPYDATA_SOURCE_DIR